dash-pipeline/dpdk-pna/README.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+At the 2023-Jan-26 DASH Behavioral Model WG meeting, I (Andy
+Fingerhut) volunteered to attempt to compile either the unmodified
+DASH P4 reference code in this directory
+
++ https://github.com/sonic-net/DASH/tree/main/dash-pipeline/bmv2
+
+using the open source p4c-dpdk compiler, or to make as few changes as
+possible to it in order for it to compile.  The goal is to determine
+whether there are any features missing in the p4c-dpdk compiler in
+order to compile this P4 code.
+
+
+# Results as of 2023-Jan-28
+
+The good news is that with only a few changes in the code, it
+compiles.
+
+There are several warning messages output by p4c-dpdk that are
+currently normal and exepcted.  These are warnings about the DPDK
```

Comment:
spelling - please check spellcheck results in https://github.com/sonic-net/DASH/actions/runs/4039001294/jobs/6943385847 and either correct errors, or add terms to `.wordlist.txt`

Reply:
Fixed